pro/src/screens/IndividualOffer/SummaryScreen/StockSection/RecurrenceSection/RecurrenceSection.tsx

```diff
@@ -15,16 +15,19 @@ const RecurrenceSection = ({
   departementCode,
 }: RecurrenceSectionProps) => {
   if (!stocksStats?.stockCount) {
-    return null
+    if (stocksStats?.stockCount !== 0) {
+      return null
+    }
   }
 
   function formatCapacity(capacity: number): string {
-    return capacity === 1 ? `${capacity} place` : `${capacity} places`
+    return capacity <= 1 ? `${capacity} place` : `${capacity} places`
   }
 
-  const totalCapacity = stocksStats.remainingQuantity
-    ? formatCapacity(stocksStats.remainingQuantity)
-    : 'Illimitée'
+  const totalCapacity =
+    stocksStats.remainingQuantity || stocksStats.remainingQuantity == 0
```

Comment:
```suggestion
    stocksStats.remainingQuantity || stocksStats.remainingQuantity === 0
```

je pense que ça change rien dans ton cas, mais c'est en général plus prudent d'utiliser des triple =,
leur comportement est un peu plus cohérent, donc tu peux le rajouter